Añadiendo archivos de abstract factory

FabioSanabria opened the pull request:
Investigación de abstract factory hecho por Fabio Sanabria y Esteban Iglesias

FabioSanabria committed:
Añadiendo archivos de abstract factory

Investigación de abstract factory hecho por Fabio Sanabria y Esteban Iglesias

FabioSanabria committed:
Agregando autoevaluación y Coevaluación

La autoevaluación y coevaluación de ambos está en el mismo excel

FabioSanabria committed:
Agregando la presentación y readme creativo

El readme es el mismo que el principal, solo que este utiliza HTML por lo que es recomendado abrirlo en Visual Studio

sivanahamer reviewed (approved):
Quiten la auto y coevaluación. Eso en mediación.

FabioSanabria committed:
Eliminando la autoevaluación y coevaluación

Subiendo los archivos a mediación virtual para su respectiva revision.

FabioSanabria commented:
Ya se eliminó la autoevaluación y coevaluación, el problema es que ya no se puede subir nada a mediación virtual

sivanahamer commented:
Revisen el error con el enlace de la presentación

FabioSanabria committed:
Eliminando el link que daba error

Se eliminó el link de la presentación que daba error y solo se sube el pdf con la presentación de Abstract Factory para que sea igual a las demás presentaciones y que no de error

FabioSanabria commented:
Ya se eliminó el link que daba el error 🐛

FabioSanabria committed:
Cambiando nombre de la carpeta

Revisando si esto es lo que ocasiona el error :bug:

FabioSanabria committed:
Eliminando Readme que daba problemas

sivanahamer commented:
No se les olvide el merge

sivanahamer commented:
Buenas,

No olviden el merge y arreglar el error.

FabioSanabria committed:
Corrijiendo error de link

FabioSanabria commented:
Profe, yo no tengo acceso al repositorio padre entonces no puedo hacer merge de las investigaciones :(
Y tengo que esperar a que apruebe los cambios

sivanahamer commented:
Hola, no pueden hacer el merge hasta que pase la prueba.

FabioSanabria committed:
Haciendo con commit para ver si está correcto
